python/gtsam/tests/test_KalmanFilter.py

```diff
@@ -19,6 +19,21 @@
 class TestKalmanFilter(GtsamTestCase):
 
     def test_KalmanFilter(self):
+
+        # Kalman Filter Definitions:
```

Comment:
Cool, but multi-line comments in python are typically done using `""" ... """`

If using VS Code, also pays to install the "docstring" extension.

Patch branch and I'll approve, merge